src/base/async/int.c

```diff
@@ -1898,8 +1898,8 @@ static int redir_it(void)
   LWORD(esi) = sda_lo; SREG(ds) = sda_hi;
   LWORD(ebx) = DOS_SUBHELPER_MFS_REDIR_INIT;
   LWORD(eax) = DOS_HELPER_MFS_HELPER;
-  if (mfs_inte6() == TRUE) {  /* Do we have a functioning redirector? */
-    redirect_devices();
+  if (mfs_inte6() == TRUE && LWORD(eax)) {
```

Comment:
Where does this come from?
Would it be possible to add the
define const to compare AX with?